cocos/renderer/gfx-gles2/GLES2Fence.h

```diff
@@ -0,0 +1,25 @@
+#ifndef CC_GFXGLES2_GLES2_FENCE_H_
+#define CC_GFXGLES2_GLES2_FENCE_H_
+
+NS_CC_BEGIN
+
+class GLES2GPUFence;
+
+class CC_GLES2_API GLES2Fence : public GFXFence {
+public:
+    GLES2Fence(GFXDevice* device);
+    ~GLES2Fence();
+
+public:
+    bool initialize(const GFXFenceInfo& info);
+    void destroy();
+    void wait();
+    void reset();
```

Comment:
It is better to add `virtual` and `override`.

Reply:
All these functions are pure virual functions

Reply:
Yep, add `virtual` to make maintaining more easier. Add `override` can let compiler to check the error. The existing implementation doesn't use it, i will modify them.